### DISCUSSION TITLE
Moving to a top level dedicated Github org

### DISCUSSION THREAD

**Discussion by pramodkvarma** [INCOMS/vc-specs#6] [Announcements]
Moved vc-specs repo into a top level Github org called [VC-Specs](https://github.com/VC-Specs). Given this org is only to enable VC models across education, skilling, and work domains, we can potentially maintain separate repos such as `core` (where common objects go), `PoWX` (proof of work, experience, rating, salary, etc), `PoES` (proof of eduction, skill, etc), and may be others as this evolve.

@kochhar @dileepbapat @parthlawate can we huddle? Suggestions?

**Comment by surendrasinghs** [comment]
Great to see this dedicated space!

As per the suggestion above, @pramodkvarma does Proof of Work and Reputation as mentioned in DESH Technical Note fall under PoWX and similarly, does Proof of Education/Training and Proof of Skill fall under PoES?
